lib/rucio/core/distance.py

```diff
@@ -141,3 +141,31 @@ def update_distances(src_rse_id=None, dest_rse_id=None, parameters=None, session
         query.update(params)
     except IntegrityError as error:
         raise exception.RucioException(error.args)
+
+
+@read_session
+def export_distances(session=None):
+    """
+    Get distances between all the rses.
+
+    :param session: The database session to use.
+
+    :returns distance: dictionary of dictionaries with all the distances.
+    """
+
+    rsemap = {}
+    for k, v in session.query(RSE.id, RSE.rse).all():
+        rsemap[k] = v
+    distances = {}
+    try:
+        query = session.query(Distance)
+        for distance in query.all():
+            src_name = rsemap[distance.src_rse_id]
+            dst_name = rsemap[distance.dest_rse_id]
+            key = src_name + '__' + dst_name
```

Comment:
A matrix in a dictionary is more intuitive, i.e, `dict[src][dst] = distances dict`